content/tutorial-x-ray-image-processing.md

```diff
@@ -0,0 +1,571 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.12
+    jupytext_version: 1.7.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Tutorial: X-ray image processing
+
++++
+
+This tutorial demonstrates how to read and process X-ray images with NumPy,
+imageio, Matplotlib and SciPy. You will learn how to load medical images, focus
+on certain parts, and visually compare them using mask filters and the
+[Sobel](https://en.wikipedia.org/wiki/Sobel_operator) and
+[Canny](https://en.wikipedia.org/wiki/Canny_edge_detector) edge detectors. X-ray
+image analysis can be part of your data analysis and
```

Comment:
@melissawm Thanks so much for driving this 👍 

Just realized something - we should mention the newly added Gaussian and Laplacian-Gaussian methods in the intro (I can open a small PR for the notebook later). Also, to improve readability for busy users, we can split the 1st paragraph into 2 shorter ones.

Suggestion:

```suggestion
This tutorial demonstrates how to read and process X-ray images with NumPy,
imageio, Matplotlib and SciPy. You will learn how to load medical images, focus
on certain parts, and visually compare them using the
[Gaussian](https://en.wikipedia.org/wiki/Gaussian_filter),
[Laplacian-Gaussian](https://en.wikipedia.org/wiki/Laplace_distribution),
[Sobel](https://en.wikipedia.org/wiki/Sobel_operator), and
[Canny](https://en.wikipedia.org/wiki/Canny_edge_detector) filters for edge
detection. 

X-ray image analysis can be part of your data analysis and
```